configs/swarm/genesis.json

```diff
@@ -90,24 +81,33 @@
         "Mint": {
           "Asset": {
             "object": "13",
-            "destination_id": "rose##alice@wonderland"
+            "destination_id": "rose##ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
           }
         }
       },
       {
         "Mint": {
           "Asset": {
             "object": "44",
-            "destination_id": "cabbage#garden_of_live_flowers#alice@wonderland"
+            "destination_id": "cabbage#garden_of_live_flowers#ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
+          }
+        }
+      },
+      {
+        "Transfer": {
+          "AssetDefinition": {
+            "source_id": "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis",
+            "object": "rose#wonderland",
+            "destination_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
           }
         }
       },
       {
         "Transfer": {
           "Domain": {
-            "source_id": "genesis@genesis",
+            "source_id": "ed01204164BF554923ECE1FD412D241036D863A6AE430476C898248B8237D77534CFC4@genesis",
             "object": "wonderland",
-            "destination_id": "alice@wonderland"
+            "destination_id": "ed0120CE7FA46C9DCE7EA4B125E2E36BDB63EA33073E7590AC92816AE1E861B7048B03@wonderland"
```

Comment:
Here `genesis.json` has a literal genesis public key, which means `kagami genesis` requires some input of the genesis public key. That's why I made `kagami genesis` read `GENESIS_PUBLIC_KEY` environment variable

Reply:
I think you already mentioned this recently: we may not need genesis account. It was recently pointed out to me by Makoto that we don't need to sign the genesis file. Instead of starting peers with the genesis public key, we can start them with hash of the genesis block. This completely removes the need to have genesis account for the purposes of signing. 

But can we do without genesis account even in genesis instructions?

Reply:
So let's reopen #4409 and discuss it there. Do you think suggested changes in this PR are ok for now?

Reply:
this should be considered in #4555